setup.py

```diff
@@ -165,8 +168,9 @@ def build_js_files(self):
         "c_string_type": "unicode",
         "c_string_encoding": "utf8",
     }
-    EXTRA_COMPILE_ARGS = ["-D_GLIBCXX_DEBUG", "-D_LIBCPP_DEBUG"]
-    UNDEF_MACROS = ["NDEBUG"]
+    if IS_LINUX:
+        EXTRA_COMPILE_ARGS = ["-D_GLIBCXX_DEBUG", "-D_LIBCPP_DEBUG"]
+        UNDEF_MACROS = ["NDEBUG"]
```

Comment:
We probably want to unset `NDEBUG` even on Darwin, I would think?

Reply:
👍